### ISSUE TITLE
Compilation Error: Rf_isValidString and Rf_isValidStringF not found in rinternals.rs

### ISSUE THREAD

**Issue by scm-mark** [extendr/extendr#847]
A compilation error seems to arise when trying to build a package that lists extendr-api, extendr-engine, libR-sys (version 0.7.0 for all) as dependencies (version rustc 1.78.0).  Part of the error is pasted below.
```
error[E0425]: cannot find function, tuple struct or tuple variant `Rf_isValidString` in this scope
     |
358  |         unsafe { Rf_isValidString(self.get()).into() }
     |                  ^^^^^^^^^^^^^^^^ help: a function with a similar name exists: `Rf_isBlankString`
     |
     |
1294 |     pub fn Rf_isBlankString(arg1: *const ::std::os::raw::c_char) -> Rboolean;
     |     ------------------------------------------------------------------------ similarly named function `Rf_isBlankString` defined here

error[E0425]: cannot find function, tuple struct or tuple variant `Rf_isValidStringF` in this scope
     |
363  |         unsafe { Rf_isValidStringF(self.get()).into() }
     |                  ^^^^^^^^^^^^^^^^^ help: a function with a similar name exists: `Rf_isBlankString`
     |
     |
1294 |     pub fn Rf_isBlankString(arg1: *const ::std::os::raw::c_char) -> Rboolean;
     |     ------------------------------------------------------------------------ similarly named function `Rf_isBlankString` defined here

For more information about this error, try `rustc --explain E0425`.
error: could not compile `extendr-api` (lib) due to 2 previous errors
```

**Comment by CGMossa** [comment]
Hello. We released a patch 0.7.1 of all of our packages to deal with it.
Please update Cargo.toml and also perhaps run cargo update, in src/rust folder.

**Comment by scm-mark** [comment]
I see, thanks!
